public/index.ejs

```diff
@@ -121,7 +117,6 @@
           <a
             class="navbar-item has-text-light"
             href="https://github.com/disease-sh/api"
-            target="_blank"
```

Comment:
This could actually use a new tab, since one might want to check out the repo and later come back? Would love to hear some thoughts

Reply:
You can always press the back button or open a new tab with the middle mouse button in the first place. I think forcing the user to open a link in a new tab is pretty annoying.

Reply:
i agree with @maltejur